protractor/local.config.ts

```diff
@@ -1,10 +1,11 @@
 import { Config,browser } from 'protractor';
-
+import { reporter } from './helpers/reporter';
 export const config: Config = {
```

Comment:
leave a blank line between the import statements and the config definition

Reply:
Done